pkg/envinfo/storage.go

```diff
@@ -119,7 +115,7 @@ func (ei *EnvInfo) ListStorage() ([]localConfigProvider.LocalStorage, error) {
 
 // DeleteStorage deletes the storage with the given name
 func (ei *EnvInfo) DeleteStorage(name string) error {
-	err := ei.devfileObj.Data.DeleteVolume(name)
+	err := ei.devfileObj.Data.DeleteVolumeMount(name)
```

Comment:
This only deletes the volume mount, you need to delete the volume component which was inherently done by the prev function.

So just need to call `DeleteComponent(name string) error`

Reply:
Addressed in c4bc9d033. PTAL.

Reply:
looks good